metadata.rb

```diff
@@ -8,8 +8,11 @@
 chef_version      '>= 15'
 version           '5.0.1'
 
-supports 'ubuntu'
-supports 'redhat'
+supports 'amazon'
+supports 'debian'
 supports 'centos'
+supports 'redhat'
+supports 'suse'
```

Comment:
Should this also include `opensuseleap`?

Reply:
Indeed it should